src/System.Diagnostics.Process/tests/RemotelyInvokable.cs

```diff
@@ -111,6 +112,22 @@ public static int ReadLineWriteIfNull()
             return SuccessExitCode;
         }
 
+        public static int ReadLineWithCustomEncodingWriteLineWithUtf8(string inputEncoding)
```

Comment:
With a little refactoring you could make this more generic `ReadAndWriteLineWithEncoding(string, string)` ?

Reply:
maybe not worth the bother

Reply:
@danmosemsft sure, but is that useful/worthwhile here? This function exists solely to support the two test methods that don't need that additional genericity.

Reply:
nope...